lib/src/core/models/kanji.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:isar/isar.dart';
+
+part 'kanji.g.dart';
+
+@collection
+@Name("Kanjis")
+@JsonSerializable()
+class Kanji {
+  final int id;
+
+  final String kanji;
+  final int? nbrTraits;
+  final int? grade;
+  @JsonKey(name: "niveau_jlpt")
+  final int niveauJLPT;
+  @Default([])
+  final List<String> traductions;
+  @Default([])
+  @JsonKey(name: "lectures_on")
+  final List<String> lecturesOn;
+  @Default([])
+  @JsonKey(name: "lectures_kun")
+  final List<String> lecturesKun;
```

Comment:
L'app mobile est en anglais pour tout ce qui est code/commentaires, c'est juste pour consistence 😅 
Petite question, je ne sais pu ce que signifie `grade`?

```suggestion
  /// Number of strokes necessary to draw the kanji
  final int? numberOfStrokes;
  final int? grade;
  @JsonKey(name: "niveau_jlpt")
  final int jlptLevel;
  @Default([])
  final List<String> translations;
  /// Pronunciations in sino-Japanese
  @Default([])
  @JsonKey(name: "lectures_on")
  final List<String> onPronunciations;
  /// Pronunciations in Japanese
  @Default([])
  @JsonKey(name: "lectures_kun")
  final List<String> kunPronunciations;
```